### PR TITLE
Pd 478 create s3 service minIO migration tutorial

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR updates the S3 Service tutorial, creates a migration tutorial for S3 Service MinIO to Enterprise MinIO, and updates the Configure Enterprise MinIO tutorial.

It implements the trueimage shortcode in the tutorial articles.

Merge this into the 22.12.3_docs branch.
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md [hunk 1]
```diff
@@ -1,69 +1,102 @@
...
 
+## Migrating from TrueNAS S3 Service MinIO
+
+When migrating from the deprecated S3 service for MinIO to the MinIO Enterprise train application, before you install the MinIO Enterprise application:
+
+* Disable the S3 service.
+  Go to **System Settings > Services** and disable the service and clear the **Start Automatically** checkbox. 
+  This prevents the service from re-enabling after a system restart.
+
+* Review your S3 service and MinIO settings and note all IP addresses, port numbers, TLS server host name, access and secret keys, storage, and certificate settings.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:91]:
```suggestion
* Review your S3 service and MinIO settings.
Note all IP addresses, port numbers, TLS server host name, access and secret keys, storage, and certificate settings.
```

**Review by DjP-iX — DISMISSED** [review]

File: content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md [hunk 2]
```diff
@@ -1,69 +1,102 @@
...
+To use the complete MinIO app without iXsystems support, see the application that is available in the Community Apps catalog.
 
 We recommend that TrueNAS SCALE Enterprise (HA) systems not deploy applications.
+
+SCALE Enterprise single controller systems with the applications and virtual machines license feature have access to the **MinIO Official Enterprise** widget. 
 {{< /enterprise >}}
 
+Community users can add this version of MinIO application by going to **Manage Catalogs**. 
+Edit the **Official** catalog by selecting **enterprise** in **Preferred Trains**, and then click **Save**.
+Both the Official Charts and Enterprise MinIO widgets display on the **Available Applications** screen.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:26]: This is incorrect. the MinIO community app is on the Available Applications screen by default. Community users can add the Enterprise version if they choose.
Enterprise customers with the correct license only see the Enterprise MinIO application on Available Applications.

[the remark this answers is not in the document]
Comment (DjP-iX) [reply content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:26]: I was thinking a link to the community version docs would be helpful if a non-enterprise user ends up here by mistake. Would it work for you with something like "Alternatively, community users can add the complete [MinIO community application]({{< relref "minioclustersscale.md" >}}), which appears on the **Available Applications** screen by default." ?

**Comment by micjohnson777** [comment]
I think this one is ready to merge, all suggested changes comitted.

File: content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md [hunk 3]
```diff
@@ -1,69 +1,104 @@
...
-**Mount Path** populates with **/data** or **/data*#***, where *#* is a number to distinguish multiple instances of MinIO such as in a cluster configuration. 
+To use an existing dataset, select **Host Path (Path that already exists on the system)**. 
+**Mount Path** populates with **/data** or **/data<em>#</em>**, where *#* is a number to distinguish multiple instances of MinIO such as in a cluster configuration. 
 Browse to the location of the dataset and click on it to populate the **Host Path**. 
 
 The remaining **minio** wizard settings are optional.
 
 Click **Save** to complete the installation.
 
+## Migrating from TrueNAS S3 Service MinIO
```
Comment (ericbsd) [inline content/SCALE/SCALETutorials/Apps/EnterpriseApps/ConfigMinioEnterprise.md:84]: This only work for on TrueNAS SCALE Enterprise.

**Pull request merged**